MediaRecorder.html

```diff
@@ -42,57 +42,29 @@
 <code>mimeType</code> specifies the container format for the recording, and that the codecs that the recording contains will normally be specified as parameters to it. </dd><dt><dfn><code>state</code></dfn> of type <span class="idlAttrType"><a>RecordingState</a></span>, readonly       </dt><dd> The current state of the MediaRecorder object.	When the MediaRecorder is created, the UA <em title="must" class="rfc2119">must</em> set this attribute to <code>inactive</code>.</dd><dt><dfn><code>onstart</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>Called to handle the start event.</dd><dt><dfn><code>onstop</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>Called to handle the stop event.</dd><dt><dfn><code>ondataavailable</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>Called to handle the dataavailable event.  Note that the Blob (see [[!FILE-API]]) of recorded data is contained in this event and can be accessed via the 'data' attribute.</dd><dt><dfn><code>onpause</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>Called to handle the pause event. </dd><dt><dfn><code>onresume</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>Called to handle the resume event. </dd><dt><dfn><code>onerror</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>Called to handle a <a>ErrorEvent</a>. </dd><dt><dfn><code>ignoreMutedMedia</code></dfn> of type <span class="idlAttrType"><a>boolean</a></span></dt><dd>If this attribute is set to <code>true</code>, the MediaRecorder will not record anything when the input Media Stream is muted.  If this attribute is <code>false</code>, the MediaRecorder will record silence (for audio) and black frames (for video) when the input MediaStream is muted.  When the MediaRecorder is created, the UA <em title="must" class="rfc2119">must</em> set this attribute to <code>false</code>.</dd><dt><dfn><code>videoBitsPerSecond</code></dfn> of type <span class="idlAttrType"><a>unsigned long</a></span>, readonly       </dt><dd>The value of the Video encoding target bit rate that was passed to the Platform (potentially truncated, rounded, etc), or the calculated one if the user has specified <code>bitsPerSecond</code>.</dd><dt><dfn><code>audioBitsPerSecond</code></dfn> of type <span class="idlAttrType"><a>unsigned long</a></span>, readonly       </dt><dd>The value of the Audio encoding target bit rate that was passed to the Platform (potentially truncated, rounded, etc), or the calculated one if the user has specified <code>bitsPerSecond</code>.</dd></dl></section><section><h2>Methods</h2><dl data-link-for="MediaRecorder" data-dfn-for="MediaRecorder" class="methods"><dt><dfn><code>start</code></dfn></dt><dd>When a <code>MediaRecorder</code> object’s <code>start()</code> method is invoked, the UA <em title="must" class="rfc2119">must</em> run the following steps:
```

Comment:
Ok, would you do the LF reintroduction, or should we add an issue?
Either way LGTM :)